dash/SQLConnection.py

```diff
@@ -0,0 +1,45 @@
+import sqlite3
+import pandas as pd
+
+class SQLConnection():
+	"""Create a SQLite connection class
+	"""
+	def __init__(self, database_file: str) -> None:
+		self.conn = sqlite3.connect(f'{database_file}')
+		self.curs = self.conn.cursor()
+	
+	def query(self, query:str) -> pd.DataFrame:
+		"""
+		query the database
+
+		Args:
+			query (str): string used to query the database
+
+		Returns:
+			pd.DataFrame: result of the query
+		"""
+		try:
+			res = pd.read_sql_query(query, self.conn)
+			print(f"Querying: '{query}'")
+			return res
+		except Exception as e:
+			print(e)
+		print('\n')
+
+	def execute(self, query:str) -> list:
```

Comment:
Do the `query` and `execute` methods need to be different? I can see one returns a DataFrame and the other a list, I assume they have different use cases?